### PR TITLE
Fixed tests

### PR DESCRIPTION
**Pull request by lewis150193**

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -1,6 +0,0 @@
-REACT_APP_APIKEY=
```
Comment (benhalverson) [inline .env.sample]: Why did you delete this file?

File: src/__tests__/App.test.js [hunk 2]
```diff
@@ -1,9 +1,10 @@
 import React from 'react';
 import { shallow } from 'enzyme';
-import App from '../index';
+import {Jsx} from "../index";
```
Comment (benhalverson) [inline src/__tests__/App.test.js]: Please explain the renaming...
Also single quotes not double

**Comment by benhalverson** [comment]
I see you are a first time contributor welcome! 🎉 @lewis150193 
I see one failing test for `Header.test.js`

File: src/__tests__/App.test.js (hunk 2 above)
Comment (lewis150193) [reply src/__tests__/App.test.js]: I made the Jsx a component from the index.js file and was testing that it AppRouter was defined. Will push a PR with singe qoutes

**Comment by lewis150193** [comment]
Thanks, I will also get on fixing the Header.test.js, This is one of my first ever open PRs :D
